lib/Logger/LogContext.h

```diff
@@ -298,8 +298,7 @@ struct LogContext::ValueBuilder<LogContext::KeyValue<K, V>, Base, Depth> {
   }
   std::shared_ptr<Values> share() && {
     return std::move(*this).passValues([]<class... Args>(Args && ... args) {
-      return std::make_shared<
```

Comment:
Unrelated to this PR, but I pulled on older version of devel into it, and clang-format complained about it.
So I had to reformat this code as well.